### PR TITLE
Resetpassword

### PR DESCRIPTION
**Pull request by ShawnCollinge**
resets password

### CODE REVIEW

File: server/custom_auth/views.py [hunk 1]
```diff
@@ -45,6 +50,26 @@ def login_view(request):
...
 
+@require_POST
+def password_reset_confirm(request, uidb64, token):
+    data = json.loads(request.body)
+    new_password = data.get('new_password', '').strip()
+
+    try:
+        uid = urlsafe_base64_decode(uidb64).decode()
+        user = User.objects.get(pk=uid)
+        print("TEST")
```
Comment (elimelt) [inline server/custom_auth/views.py]: print in prod

File: server/members/views.py [hunk 2]
```diff
@@ -195,3 +198,17 @@ def post(self, request, *args, **kwargs):
...
+    permission_classes = [IsApiKey]
+
+    def post(self, request, *args, **kwargs):
+        discord_id = request.data.get("discord_id")
+        user = get_object_or_404(User, discord_id=discord_id)
+
+        token = default_token_generator.make_token(user)
+        uid = urlsafe_base64_encode(force_bytes(user.pk))
+
+        reset_url = f"https://interview.swecc.org/password-reset-confirm/{uid}/{token}/"
```
Comment (elimelt) [inline server/members/views.py:212]: can we make this work with local dev too?

File: server/custom_auth/views.py (hunk 1 above)
Comment (elimelt) [reply server/custom_auth/views.py]: ah it's outdated now

**Review by elimelt — APPROVED** [review]
LGTM besides client URL not being local dev friendly

File: server/members/views.py (hunk 2 above)
Comment (ShawnCollinge) [reply server/members/views.py:212]: Ugh, yeah I'll add that later. Tired af and want to let sophia reset password.

**Pull request merged**